extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -194,6 +220,50 @@ function renderPassiveFeedback(video, response) {
             }
         }
     });
+
+    let keywordsMap = new Map();
+
+    labels.forEach((timestamp) => {
+        getVideoContextKeyWords(keywordsMap, video, timestamp);
+    });
+
+    let canvas = document.getElementById(`passiveFeedback_${video.videoID}`);
+    canvas.onclick = function(evt) {
+        let firstPoint = chart.getElementAtEvent(evt)[0];
+
+        if (firstPoint) {
+            let label = chart.data.labels[firstPoint._index];
+            console.debug(label, keywordsMap[label]);
+            displayContextKeyWords(label, keywordsMap);
+        }
+    };
+}
+
+async function getVideoContextKeyWords(keywordsMap, video, timestamp) {
+    keywordsMap[timestamp] = false;
```

Comment:
This is a very bad practice; mixing the types of values in a map. I honestly don't think you need this; JavaScript will return `null` or `undefined` if the key has not been seen before. `if` statements will treat them as false